assets/js/components/Battery/BatterySettingsModal.vue

```diff
@@ -369,10 +368,10 @@ export default defineComponent({
 			if (!Array.isArray(this.battery)) {
```

Comment:
```suggestion
			if (!this.battery) {
```

Der hier müsste noch rein. Die Suggestion weiter oben ist wahrscheinlich irgendwie untergegangen.